.gitlab-ci.yml

```diff
@@ -133,6 +149,7 @@ build:orig:debian:bullseye:amd64:
   before_script:
     - *dind-login
   script:
+    - *requires-docker
```

Comment:
Instead of having this and the retry exit codes in the two jobs, we could now have a base job like:

```
.build:base:
  before_script:
    - *dind-login
    - *requires-docker
  retry:
    max: 2
    exit_codes:
      - 137
      - 192
```

And then have both `build:orig:debian:bullseye:amd64` and `build:packages:cross` extend it.

Reply:
sure. so, now we are only one step away from moving `.build:base:` to a template? :>